.travis.yml

```diff
@@ -21,8 +21,8 @@ jobs:
       script: npm run test
 
     - stage: Integration tests
-      name: "Truffle 5.x"
-      script: ./test-integration/simple-project-truffle-5.x/run.sh
+      name: "Integration tests"
+      script: npm run test-integration
```

Comment:
Hm, I had set it up so that each test was a single job, allowing us to easily see that they were all being run, passing, etc. Something like

```
- stage: Integration tests
  name: "Truffle 5.x"
  script: ./test-integration/simple-project-truffle-5.x/run.sh

- stage: Integration tests
  name: Ganache 2.2.x
  script: ./test-integration/ganache-2.2.x/run.sh
```